src/foraging_gui/Foraging.py

```diff
@@ -736,13 +765,32 @@ def _GetSettings(self):
             'default_ui':'ForagingGUI.ui'
         }
         
+        # Try to load Settings_box#.csv
+        self.SettingsBox={}
+        try:
+            if os.path.exists(self.SettingsBoxFile):
```

Comment:
Why load the csv as a dataframe only to convert to a dictionary? Lets just load directly as a dictionary, and avoid importing the pandas package

Reply:
We have already installed the pandas package. How to directly load the csv as a dictionary?

Reply:
https://stackoverflow.com/questions/6740918/creating-a-dictionary-from-a-csv-file

Reply:
Ok, thanks for sharing. I think it's ok to use pandas here as we already have that package installed.

Reply:
Why not update the code to use csv straight to a dictionary for readability

Reply:
I think the pandas is doing the same thing.

Reply:
Hey Alex, you prefer the csv.reader() instead of the pandas.read_csv()?

Reply:
Or at least doing something like this: 
```
SettingsBox = {row[0]:row[1] for rows in df}
```
instead of 

```                
for idx, row in df.iterrows():
    key = row.iloc[0]  # First column
    value = row.iloc[1]  # Second column
    self.SettingsBox[key] = value
```

Reply:
I replaced it with `self.SettingsBox = {row[0]: row[1] for _, row in df.iterrows()}`